hetzner/README-cloud.md

```diff
@@ -91,6 +96,29 @@ It's used so Ansible can connect to the server without requiring password.
 
 # The VM
 
+## All-in-one creation
+
+2 new playbooks have been created to aggregate the different operations executed by the different playbooks
+, `hetzner-create-server-aggregate.yml` and `hetzner-delete-server-aggregate.yml`.
+
+To create and deploy a new VM simply execute the following command on the root of the `k8s-infra` project.
+
+```bash
+$ ansible-playbook hetzner/ansible/hetzner-create-server-aggregate.yml -e vm_name=my_vm -e k8s_type=masters -e k8s_version=119 -e salt_text=$(gpg --gen-random --armor 1 20)  --tags create
```

Comment:
Why dont you use as playbook name `create-vm-and-k8s.yml` ?

Reply:
because this playbook only creates the VM server on hetzner, it doesn't deploy k8s on that server.

Reply:
ok.